src/tools/device.cpp

```diff
@@ -535,4 +535,4 @@ void Device::cmdDisplay() {
   std::cout << name << ": " << state_ << endl
             << "sanityCheck: " << sanityCheck_ << endl
             << "controlInputType:" << controlInputType_ << endl;
-}
+}
```

Comment:
Please avoid this type of modification that pollutes the history.

Reply:
This probably due to a trailing whitespace.

Reply:
No, this is due to a lack of newline at the end of file.

Reply:
Sorry I didn't see that I removed the last line I will correct it with the other remarks.